### PR TITLE
Get all users for a space or organization

### LINKED ISSUE #224: Get all users implementation missing
Hey, it looks like contentful-management.rb is missing the get all users API feature, which is quite critical for some user access control synchronization work our team at Square is about to execute on

API Reference: https://www.contentful.com/developers/docs/references/user-management-api/#/reference/users

Ruby SDK missing `.all` method/functionality: 
image

Any ETA on this missing functionality being added?

### PR DESCRIPTION
**Pull request by rubydog**
Adds ability to retrieve all users in a space or organisation. Please refer to https://www.contentful.com/developers/docs/references/user-management-api/#/reference/users for more details.

Fixes #224

### CODE REVIEW

File: lib/contentful/management/version.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@ module Contentful
   # Management Namespace
   module Management
     # Gem Version
-    VERSION = '3.0.0'.freeze
+    VERSION = '3.0.1'.freeze
```
Comment (dlitvakb) [inline lib/contentful/management/version.rb:6]: Version bump should be done on the release commit, also it should be `3.1.0` as this is a minor version bump, and not a bugfix.

**Review by dlitvakb — CHANGES_REQUESTED** [review]
Needs docs changes, code is good

**Review by dlitvakb — APPROVED** [review]

File: lib/contentful/management/organization_user_methods_factory.rb [hunk 2]
```diff
@@ -13,6 +13,10 @@ def initialize(client, organization_id)
       def find(id)
         User.find(client, nil, nil, id, @organization_id)
       end
+
+      def all
```
Comment (ghepting) [inline lib/contentful/management/organization_user_methods_factory.rb:17]: this method needs to accept parameters such as limit, etc as the other .all interfaces across other resources types accept.

Comment (rubydog) [reply lib/contentful/management/organization_user_methods_factory.rb:17]: @ghepting thanks for flagging it. I will take a look at it today.

Comment (ghepting) [reply lib/contentful/management/organization_user_methods_factory.rb:17]: Great, thanks again for your efforts here. Really want to see these ruby SDKs become more first-class citizens in the Contentful developer toolkits  :)

**Pull request merged**